### DISCUSSION TITLE
Update Hotspots on changing model

### DISCUSSION THREAD

**Discussion by AR13202** [google/model-viewer#4684] [Q&A]
Hello, Is there any way to update hotspots in animated model. I want to update the data-surface attribute on changing model.

Thanks

**Comment by elalish** [comment]
Yes, but it looks like our docs are out of date. This method works: https://modelviewer.dev/docs/index.html#entrydocs-annotations-methods-updateHotspot, but use `{name, surface}` as the structure members.

**Reply by elalish** [reply]
update on the way...
